packages/komodo_defi_framework/lib/src/operations/kdf_operations_local_executable.dart

```diff
@@ -77,7 +77,7 @@ class KdfOperationsLocalExecutable implements IKdfOperations {
       // Store the coins list in a temp file to avoid command line argument and
       // environment variable value size limits (varies from 4-128 KB).
       // Pass the config directly to the executable as an argument.
-      final tempDir = await Directory.systemTemp.createTemp('mm_coins_');
+      final tempDir = await getTemporaryDirectory();
```

Comment:
```suggestion
      final tempDir = (await getTemporaryDirectory()).createTemp('mm_coins_');
```

While having multiple instances running isn't part of our current use case, this avoids the slight chance of potential conflicts.